src/index.ts

```diff
@@ -44,13 +77,20 @@ export * from './errors';
  * @throws {StrapiInitializationError} If the provided baseURL doesn't conform to a valid HTTP or HTTPS URL,
  *                                        or if the auth configuration is invalid.
  */
-export const strapi = (config: StrapiConfig) => {
-  const configValidator = new StrapiConfigValidator();
-
-  return new Strapi<typeof config>(
-    // Properties
-    config,
-    // Dependencies
-    configValidator
-  );
+export const strapi = (config: Config) => {
+  const { baseURL, auth } = config;
+
+  const sdkConfig: StrapiConfig = { baseURL };
+
+  // In this factory, to keep things simple, users can't manually set the strategy options.
```

Comment:
```suggestion
  // In this factory, while there is only one auth strategy available, users can't manually set the strategy options.
```
Is this more accurate?

Reply:
Well it's a bit of both, I don't mind updating though :slightly_smiling_face: